contributor.md

```diff
@@ -0,0 +1,5 @@
+Replaced now()->toDateTimeString() with Carbon::now()
+
+I got the below error so changed below lines.
+Uncaught exception 'Error' with message 'Call to undefined function Anik\ElasticApm\Middleware\now()' 
+in /var/www/vendor/anik/elastic-apm-php/src/Middleware/RecordForegroundTransaction.php:87
```

Comment:
Is this file really necessary?

Reply:
No. You can remove that. I added for you to know what is the issue.

Reply:
So you can suggest me the changes needed.

Reply:
Removed that file

Reply:
Thanks.